overlays/custom-packages/default.nix

```diff
@@ -13,5 +13,6 @@ _: {
     (import ./qemu)
     (import ./nm-launcher)
     (import ./labwc)
+    (import ./tcp-ssh)
```

Comment:
We don't necessarily need to circle this stuff through overlay

Reply:
If i remove the custom-packages part the build of lenove-x1-carbox.nix cannot find the package name.